src/app/layout/components.barrel.ts

```diff
@@ -0,0 +1,51 @@
+// Import components and services etc here
+
+import { HttpClientModule } from "@angular/common/http";
+import { HttpModule } from "@angular/http";
+
+
+import {
+    MatToolbarModule, MatButtonModule, MatIconModule, MatMenuModule, MatSidenavModule,
```

Comment:
Can you please use the material module to import everything from Angular material.

Reply:
import {
    MatToolbarModule, MatButtonModule, MatIconModule, MatMenuModule, MatSidenavModule,
    MatListModule, MatChipsModule, MatProgressBarModule, MatCardModule, MatTabsModule
} from '@angular/material';

You can see that i'm importing material modules from Angular material.
If i misunderstood this point please explain it with some details.